.gitignore

```diff
@@ -2,3 +2,111 @@ build_and_deploy/node_modules
 .idea
 build_and_deploy/**/*.js
 build_and_deploy/**/*.js.map
+dist
+dist/index.js
```

Comment:
Why are we adding dist and dist/index.js to gitignore? The action runs from this file